script/generate_dispatch.py

```diff
@@ -287,7 +288,7 @@
 
 header = license + info + body
 
-header_file = open("../src/VkBootstrapDispatch.h", "w")
+header_file = codecs.open("../src/VkBootstrapDispatch.h", "w", "utf-8")
```

Comment:
Not thinking anything is wrong, but am curious to know what all this does and why it is necessary? (I can guess why but it'd be good to hear the reason for posterity's sake)

Reply:
The script was generating the source file in Windows-1252 encoding when I ran it, which makes me think that `open` defaults to the platforms default encoding when creating new files and not opening in binary. The change just makes sure the output file is in UTF-8.

Reply:
Ah that error, yeah this is a good idea then.